packages/pigeon/lib/dart_generator.dart

```diff
@@ -512,12 +501,12 @@ $resultAt != null
 /// available for dependency injection.  If it is left null, the default
 /// BinaryMessenger will be used which routes to the host platform.
 ${api.name}({BinaryMessenger? binaryMessenger})
-\t\t: _binaryMessenger = binaryMessenger;
-final BinaryMessenger? _binaryMessenger;
+\t\t: ${_varNamePrefix}binaryMessenger = binaryMessenger;
+final BinaryMessenger? ${_varNamePrefix}binaryMessenger;
 ''');
 
-      indent
-          .writeln('static const MessageCodec<Object?> codec = $codecName();');
+      indent.writeln(
+          'static const MessageCodec<Object?> pigeonChannelCodec = $codecName();');
```

Comment:
We should make a file-level private constant for `pigeonChannelCode` since it's in a few places.